app/views/matches/show.html.erb

```diff
@@ -0,0 +1,63 @@
+<div>
+  <h4><%= @my_item.name %> &amp; <%= @other_item.name %></h4>
+<!--   <%= render @other_item %> -->
+
+  <%= form_for @my_item, method: "PATCH" do |f| %>
```

Comment:
Rails will know this is a patch as long as my_item is saved. You don't need to tell it.